
export async function signup(c:any){
    try{
        return c.text("Hello")
    }catch(error){
        console.log("Error occured in signup");
    }   
}

export async function signin(c:any){
    try{

    }catch(error){
        console.log("Error occured in singin");
    }
}